src/lib.rs

```diff
@@ -1,15 +1,27 @@
 //! Tools for finding and manipulating differences between files
 //!
-//! ## Overview
+//! # Overview
 //!
 //! This library is intended to be a collection of tools used to find and
-//! manipulate differences between files inspired by [LibXDiff] and [GNU
-//! Diffutils]. Version control systems like [Git] and [Mercurial] generally
-//! communicate differences between two versions of a file using a `diff` or
-//! `patch`.
+//! manipulate differences between files or arbitrary data inspired by
+//! [LibXDiff] and [GNU Diffutils]. Version control systems like [Git] and
+//! [Mercurial] generallycommunicate differences between two versions of a
```

Comment:
lacking space between words

Reply:
Fixed. I've also rebased so that this branch no longer conflicts with `master`